presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -174,6 +191,105 @@ public void testBuildSqlWithString()
     }
 
     @Test
+    public void testBuildSqlWithDateTime()
```

Comment:
Same comment as on the varchar commit. Either merge these into the same test, or only include values for the specific type being tested.